build.sbt

```diff
@@ -8,3 +8,5 @@ addSbtPlugin("com.github.gseitz" % "sbt-release" % sbtReleaseVersion)
 addSbtPlugin("com.jsuereth" % "sbt-pgp" % sbtPgpVersion)
```

Comment:
You need to use:

```scala
 addSbtPlugin("com.jsuereth" %% "sbt-pgp" % sbtPgpVersion)
```
Double `%` so that the correct sbt/scala versions will be used when resolving the dependency.